src/client/Microsoft.Identity.Client/WsTrust/CommonNonInteractiveHandler.cs

```diff
@@ -137,8 +137,8 @@ public async Task<UserRealmDiscoveryResponse> QueryUserRealmDataAsync(string use
             {
                 throw new MsalClientException(
                     MsalError.ParsingWsTrustResponseFailed,
-                    "There was an error parsing WS-Trust response from the endpoint. This may occur if there is an issue with your ADFS configuration." 
```

Comment:
Agreed with what @pmaytak said - https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/WsTrust/WsTrustWebRequestManager.cs#L103 does not seem to log anything here. We should log the ws-trust response in case of error, but as PII.

Reply:
Added logs